addon/styles/_pix-collapsible.scss

```diff
@@ -0,0 +1,73 @@
+.pix-collapsible {
+  @import 'reset-css';
+
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  font-family: $font-roboto;
+  font-size: 1rem;
+  border: 1px solid $grey-20;
+  padding: 0;
+  cursor: pointer;
+  background-color: $white;
+
+  &__title {
+    margin: 0;
+    box-sizing: border-box;
+    padding: 14px 10px 14px 16px;
+    min-width: 100%;
+    cursor: pointer;
+    display: flex;
+    justify-content: space-between;
+    background: transparent;
+    border: none;
+    color: $grey-60;
+    font-size: 1rem;
+    font-weight: 500;
+
+    .pix-collapsible-title__icon {
+      color: $grey-45;
+      margin-right: 6px;
+    }
+
+    &:hover {
+      background-color: $grey-10;
+    }
+
+    &:focus {
+      background-color: $grey-10;
+    }
+
+    &--uncollapsed {
+      background-color: $grey-10;
+    }
+  }
+  
+  &__content {
+    padding: 16px 20px;
+    display: flex;
+    align-items: center;
+  }
+}
+
+.pix-collapsible:first-child {
+  margin-top: 0;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.pix-collapsible:last-child {
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.pix-collapsible--uncollapsed {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  box-shadow: 0 2px 5px 0 rgba($black, 0.05);
+  border-radius: 4px;
+}
+
+.pix-collapsible:not(.pix-collapsible--uncollapsed) + .pix-collapsible:not(.pix-collapsible--uncollapsed) {
```

Comment:
🤯 je comprend pas mais ça a l'air classe haha

Reply:
Alors je traduit : 
`.pix-collapsible:not(.pix-collapsible--uncollapsed)` : ça sélectionne les élément qui ont la classe `pix-collapsible` mais pas la classe `pix-collapsible--uncollasped` 
`+ .pix-collapsible:not(.pix-collapsible--uncollapsed)` : ça sélectionne le frère direct de l'élément précédent, ce frère direct est de la classe `pix-collapsible` mais pas `pix-collapsible--uncollapsed`

Autrement dit : si tu as 2 éléments plié d'affilé, tu enlèves la bordure au second :) 

Sans cette régle ça donne une double bordure entre les éléments : 
<img width="189" alt="image" src="https://user-images.githubusercontent.com/38167520/118952170-01027480-b95c-11eb-8421-801f52b144d8.png">

Reply:
C'est bien plus clair, merci !